docs/development/core/server/kibana-plugin-server.elasticsearcherrorhelpers.md

```diff
@@ -4,7 +4,7 @@
 
 ## ElasticsearchErrorHelpers class
 
-Helpers provided to simplify future migration away from Boom as internal Error.
+Helpers for working with errors returned from the Elasticsearch service.Since the internal data of errors are subject to change, consumers of the Elasticsearch service should always use these helpers to classify errors instead of checking error internals such as `body.error.header[WWW-Authenticate]`

```

Comment:
That's not related to changes, but docs are generated only when api changed and any updates in comments/docs are ignored.